images/disks-images-provider/entrypoint.sh

```diff
@@ -45,6 +45,16 @@ rm /images/fedora-cloud/disk.qcow2
 
 echo "copy all images to host mount directory"
 cp -R /images/* /hostImages/
+echo "make the alpine image ready for parallel use"
+cp -r /hostImages/alpine hostImages/alpine1
+cp -r /hostImages/alpine hostImages/alpine2
+cp -r /hostImages/alpine hostImages/alpine3
```

Comment:
is there any practical way to parameterize the number of workers so we don't get surprised in the future by this when someone changes the test worker count?

Reply:
I can check it. We may be able to inject an environment var with the number of workers. Need to reexamine the exact setup flow.The mist tricky part is, that this is done on `cluster-sync`, whereas the number of workers can change on every `make functest`.

Reply:
Saving this task for later. We will not increase it short-term and it is pretty hard to change in the scope of this PR.